apps/web/public/static/locales/en/common.json

```diff
@@ -2613,5 +2613,7 @@
   "limit_team_booking_frequency_description": "Limit how many times members can be booked across all team event types (collective and round-robin)",
   "booking_limits_updated_successfully": "Booking limits updated successfully",
   "you_are_unauthorized_to_make_this_change_to_the_booking": "You are unauthorized to make this change to the booking",
+  "hide_calendar_event_details": "Hide calendar event details",
+  "description_hide_calendar_event_details": "Depending on your calendar, will set calendar event details to private",
```

Comment:
This is pretty non-descriptive. Where do we plan to tell them which calendars this is supported on?

Reply:
Agree this isn't the best approach, right now we will support ICS, iCalendar, Apple Calendar, Office 365 Calendar, Outlook Calendar, CalDav and Google Calendar. 

With that range of calendars the question is raised whether we want to mention "limited" calendar support at all.